src/main/java/com/jc/tm/database/entity/Task.java

```diff
@@ -6,19 +6,19 @@
 import java.util.List;
 
 public class Task {
-    private int id;
+    private long id;
```

Comment:
for avoid addition autoboxing pls use Long instead long

Reply:
It is just mistake, because in other classes I use Long.